### PR TITLE
Clean up config values and vaults.json file.

### PR DESCRIPTION
**Pull request by pwei1018**
*Issue #:*
https://github.com/bcgov/entity/issues/6076

*Description of changes:*

1. Move vaults setting from github actions to project folder;
2. Clean up the config values that match the vault;

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of the sbc-pay license (Apache 2.0).

### CODE REVIEW

File: pay-api/src/pay_api/config.py [hunk 1]
```diff
@@ -133,19 +130,22 @@ class _Config():  # pylint: disable=too-few-public-methods
...
+    SENTRY_DSN = _get_config('SENTRY_DSN', default=None)
+
     # Valid Payment redirect URLs
     VALID_REDIRECT_URLS = [(val.strip() if val != '' else None)
                            for val in _get_config('VALID_REDIRECT_URLS', default='').split(',')]
 
     # Service account details
-    KEYCLOAK_SERVICE_ACCOUNT_ID = _get_config('KEYCLOAK_SERVICE_ACCOUNT_ID')
-    KEYCLOAK_SERVICE_ACCOUNT_SECRET = _get_config('KEYCLOAK_SERVICE_ACCOUNT_SECRET')
+    KEYCLOAK_SERVICE_ACCOUNT_ID = os.getenv('SBC_AUTH_ADMIN_CLIENT_ID')
```
Comment (saravanpa-aot) [inline pay-api/src/pay_api/config.py:147]: I hope its ok to change from _get_config to os.getenv

**Review by saravanpa-aot — APPROVED** [review]
do you know why tests are failing?

File: jobs/payment-jobs/tests/jobs/test_cfs_create_account_task.py [hunk 2]
```diff
@@ -124,5 +124,5 @@ def test_update_pad_account(session):
     assert updated_cfs_account.bank_number == new_payment_details.get('bankInstitutionNumber')
 
     assert cfs_account.status == CfsAccountStatus.INACTIVE.value
-    assert updated_cfs_account.status == CfsAccountStatus.PENDING_PAD_ACTIVATION.value
+    assert updated_cfs_account.status == CfsAccountStatus.ACTIVE.value
```
Comment (saravanpa-aot) [inline jobs/payment-jobs/tests/jobs/test_cfs_create_account_task.py:127]: this test case has to be rechecked...

Comment (sumesh-aot) [reply jobs/payment-jobs/tests/jobs/test_cfs_create_account_task.py:127]: Yes. I will run in local to check this.

Comment (sumesh-aot) [reply jobs/payment-jobs/tests/jobs/test_cfs_create_account_task.py:127]: This is correct. We moved the logic to update the bank information right away if PAD information is getting updated, which will result in ACTIVE cfs_account.

**Review by sumesh-aot — APPROVED** [review]

File: pay-api/src/pay_api/config.py (hunk 1 above)
Comment (pwei1018) [reply pay-api/src/pay_api/config.py:147]: Fixed

Comment (pwei1018) [reply pay-api/src/pay_api/config.py]: fixed

**Pull request merged**